core/src/main/java/hudson/model/Actionable.java

```diff
@@ -53,7 +53,7 @@
      *
      * Typed more strongly than it should to improve the serialization signature.
      */
-    private volatile CopyOnWriteArrayList<Action> actions;
+    private volatile CopyOnWriteArrayList<Action> actions = new CopyOnWriteArrayList<Action>();
```

Comment:
@oleg-nenashev You mean there are object where you will never call getActions? Adding that line was to further reduce the number of locks, but we can safely remove that line change and leave the rest.